apps/jubmoji-quest/prisma/schema.prisma

```diff
@@ -11,5 +11,63 @@ datasource db {
 }
 
 model Card {
```

Comment:
@AndrewCLu what about the card images and emojis? will be stored here?

Reply:
@kalidiagne I think the split between what is stored here versus in the hardcoded file is - "What attributes of the card are fixed in hardware?" So by this criteria the cardImagePath would be stored in the database and the emojiUnicode would be stored in the file.

Reply:
@vb7401 thoughts?

Reply:
I see, what if`cardImagePath` and `emojiUnicode` are both stored in the file instead?

Reply:
The `emojiUnicode` we can definitely store in the file. We can store `cardImagePath` in a file as well, but it should be within `jubmoji-quest`, not `jubmoji-api`. The api should contain data refers to the hardware itself which will always be constant, but imagePath is a frontend concept.

Reply:
> The emojiUnicode we can definitely store in the file. We can store cardImagePath in a file as well, but it should be within jubmoji-quest, not jubmoji-api. The api should contain data refers to the hardware itself which will always be constant, but imagePath is a frontend concept.

This makes sense to me; `imagePath` is indeed a fully frontend concept, so let's put it in `jubmoji-quest`. And let's put `emojiUnicode` into hardcoded public file. 

Another way I think about the hardcoded public file is: what would be put in a smart contract if we wanted to decentralize this? In my mind that's only the pubKey -> emoji mapping, and maybe a hash commitment to the other details. But everything else would live in DB